middlewares/validate.js

```diff
@@ -0,0 +1,58 @@
+import { body, validationResult } from 'express-validator';
+
+export const validateSignUpForm = [
+    body('email')
+        .exists()
+        .withMessage('email is required')
+        .isLength({ min: 1 })
+        .withMessage('email is required')
+        .isEmail()
+        .withMessage('email field must contain a valid email address')
+        .trim(),
+
+    body('firstName')
+        .exists()
+        .withMessage('firstName field is require')
+        .isLength({ min: 1 })
+        .withMessage('firstName field is require')
+        .isString()
+        .withMessage('the name must be a string')
+        .trim(),
+
+    body('lastName')
+        .exists()
+        .withMessage('lastName field is required')
+        .isLength({ min: 1 })
+        .withMessage('lastName field is required')
+        .isString()
+        .withMessage('the name must be a string')
+        .trim(),
+
+    body('passwordHash')
```

Comment:
Change name here to `password`. The value coming from postman should be `password` and not `passwordHash`